### DISCUSSION TITLE
Question about the example code on the website that use A.compose

### DISCUSSION THREAD

**Discussion by kronee0516** [albumentations-team/albumentations#1270] [Q&A] [answered]
So i saw this code on the website

```
transform = A.Compose([
    A.RandomCrop(width=450, height=450),
    A.HorizontalFlip(p=0.5),
    A.RandomBrightnessContrast(p=0.2),
], bbox_params=A.BboxParams(format='coco'))
```

but I can't use all the augmentation on this page [](https://albumentations.ai/docs/getting_started/transforms_and_targets/ )
some of the parameters are not match too

for example
the channel_shuffle , it needs img and channel_shuffled
but the site just stated p
i noticed that they have different names one is channel_shuffle other one is ChannelShuffle

and AdvancedBlur
it shows 'module 'albumentations' has no attribute 'AdvancedBlur' '

how do i use all the augmentations with A.compose?
or it is not available?

**Comment by Dipet** [comment, marked as answer]
Please provide a link to the example you are talking about.

> but the site just stated p

Augmentations have default arguments.

> it shows 'module 'albumentations' has no attribute 'AdvancedBlur' '

Try to update albumentations version: `pip install --upgrade  albumentations`

**Reply by kronee0516** [reply]
thanks for the answer

here is the link
[https://albumentations.ai/docs/getting_started/transforms_and_targets/](https://albumentations.ai/docs/getting_started/transforms_and_targets/)

I just checked i haven't upgrade the albumentation
my bad
